sos/report/plugins/salt.py

```diff
@@ -26,12 +27,46 @@ def setup(self):
         else:
             self.add_copy_spec("/var/log/salt")
 
-        self.add_copy_spec("/etc/salt")
-        self.add_forbidden_path("/etc/salt/pki/*/*.pem")
+        self.add_copy_spec([
+            "/var/log/venv-salt-minion.log",
+            "/var/log/salt-ssh.log",
+        ])
+
+        self.add_copy_spec([
+            "/etc/salt",
+            "/etc/venv-salt-minion/",
+            "/usr/local/etc/salt",
+        ])
+        self.add_forbidden_path([
+            "/etc/salt/pki/*/*.pem",
+            "/etc/venv-salt-minion/pki/*/*.pem",
+            "/usr/local/etc/salt/pki/*/*.pem",
+        ])
+
+        self.add_copy_spec([
+            "/etc/machine-id",
+            "/var/lib/dbus/machine-id",
```

Comment:
We collect it in `dbus` plugin already, while our aim is different plugins collect different data.

Is there some reason to keep this here? Or reversely, does it make sense if `dbus` plugin collects also `/etc/machine-id`? (not that `sos` collecting `/var/lib/dbus/machine-id` - which is usually a symlink to `/etc/machine-id` - does collect also the symlink destination, so there is no explicit need to collect it - until the symlink is not correct..?)

Reply:
Ah my bad, I was testing this with the `-k` option to execute only the salt plugins. 

Re `/etc/machine-id`, I found it in the documentation as the primary location to the alternative `/var/lib/dbus/machine-id`  location, so I simply included both. I see that the `dbus` plugin already collects the file as you say, so I removed it from this PR.